setup.cfg

```diff
@@ -1,3 +1,100 @@
+[metadata]
+name = desitemplate
+version = attr: desitemplate.__version__
+author = DESI Collaboration
+author_email = desi-data@desi.lbl.gov
+license = BSD 3-Clause License
+license_files = LICENSE.rst
+url = https://github.com/desihub/desitemplate
+description = DESI template package
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+edit_on_github = True
+github_project = desihub/desitemplate
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering :: Astronomy
+    Topic :: System :: Software Distribution
+    Topic :: Utilities
+
+[options]
+zip_safe = True
+package_dir =
+    =py
+packages = find:
+include_package_data = True
+python_requires = >=3.8
+; setup_requires = setuptools_scm
+; install_requires =
+;     requests
+;     pyyaml
+;     numpy<1.23
+;     astropy>=5.0
+;     healpy
+;     matplotlib
+scripts =
+    bin/template_main_script
+
+; [options.entry_points]
+; console_scripts =
+;     template_main_script = desitemplate.main:main
+
+[options.extras_require]
+test =
+    pytest
+coverage =
+    pytest-cov
+    coveralls
+doc =
+    Sphinx
+    sphinx-rtd-theme
+
+[options.package_data]
+; desitemplate = data/*
+; desitemplate.test = t/*
```

Comment:
Editorial, no action needed here: this was one of our most common failure modes for the previous setup.py -- adding a data subdirectory but forgetting to also add it to setup.py and thus having it work for development checkouts but not for tagged and installed code.  Using permissive globs like this is good.